sdk/core/azure-core-opentelemetry/inc/azure/core-opentelemetry/dll_import_export.hpp

```diff
@@ -0,0 +1,60 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// SPDX-License-Identifier: MIT
+
+/**
+ * @file
```

Comment:
Why the include directory is `azure/core-opentelemetry/`, and not `azure/core/opentelemetry/`?

Reply:
azure-core-opentelemetry is a separate package from azure-core. We cannot put any opentelemetry related files under azure/core/azure-core.

The distributed tracing folks require that the actual tracing implementation be implemented as a separate package from azure core.

Reply:
azure-core-tracing-opentelemetry is a separate package from azure-core. 

I don't know how to put folders in azure/core/internal into a package that isn't a part of azure-core.

Reply:
It can be a separate package and have include folder `inc/azure/core/opentelemetry`: multiple packages do have `inc/azure` folder, and they are combined into a single tree, none of them replaces other.